test/drag/drag.js

```diff
@@ -1,5 +1,5 @@
-//load a lot of stuff ...
-steal(function(){
+steal("../../browser/jquery", "../../node_modules/jquerypp/index", function($){
+	window.jQuery = $;
```

Comment:
Weeelll... When I said it's not needed, I guess I was a littler overly confident.  That test breaks without it, so it *is* needed.  Why it breaks, though, is not clear to me..